app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSecurity.tsx

```diff
@@ -1,109 +1,252 @@
 import {
   Alert,
+  Checkbox,
   Form,
   FormGroup,
-  Radio,
+  FormSelect,
+  FormSelectOption,
   Stack,
   StackItem,
   TextInput,
   Title,
 } from '@patternfly/react-core';
 import * as React from 'react';
 import { toValidHtmlId } from '../../../helpers';
+import { ICreateConnectorPropsUi } from './ApiConnectorCreatorSecurityForm';
 
-export interface IAuthenticationTypes {
+export interface IDropdownOption {
   value?: string;
   label?: string;
 }
 
+export interface IDropdownOptions {
+  [key: string]: IDropdownOption[];
+}
+
+export interface II18n {
+  [key: string]: string;
```

Comment:
If you do want to use this interface, I'd suggest adding all the keys that are used in the component so that the consumer of this component has type info.